lua/incrementNonce.lua

```diff
@@ -0,0 +1,21 @@
+--[[
+Input:
+KEYS[1] nonce key
+ARGV[1] current nonce
+Output:
+N OK (current nonce)
+]]
+local nonceKey = KEYS[1]
+local currentNonce = tonumber(ARGV[1])
+local rcall = redis.call
+
+local nonce = rcall("GET",nonceKey)
```

Comment:
should be check for existence of nonceKey in code `if nonce == false then`

Reply:
It does check that in the if on line 13th. If the key does not exist `not nonce` condition will satisfy it
`if not nonce or tonumber(nonce) < currentNonce then`